### DISCUSSION TITLE
New Colors and more

### DISCUSSION THREAD

**Discussion by Mogli12** [Mogli12/KeyboardSteer#775] [Announcements]
I changed the colors in HUD.
* blue color means active
* white color is normal or inactive

The gear text got a small background and appears below the speed meter. If the color is blue, then manual shifting is enabled. If the color is white, then the vehicle shifts automatically.

The inactive lines for guidance steering are not white as well. 

How do these colors work for you? Any other ideas for improvements?

**Comment by janis1405** [comment]
For me personally it is a great improvement. Maybe the inactive state could be a little more transparent for my taste.
But thank you very much for the change.

**Comment by Mogli12** [comment]
Good point. I will try to change transparency.
